core/domain/exp_services.py

```diff
@@ -807,6 +807,9 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
 
     exploration.version += 1
 
+    # Update summary of changed exploration.
```

Comment:
I'm having a lot of trouble with this-- @seanlip , can we make a time to chat sometime soon? I'm available all this evening (Friday).